run_blockchain_terraformer.py

```diff
@@ -75,11 +106,27 @@
 CARBON_POL_NAME = "bancor_pol"
 SHIBA_V2_NAME = "shiba_v2"
 SCALE_V2_NAME = "scale_v2"
+EQUALIZER_V2_NAME = "equalizer_v2"
 SOLIDLY_V2_NAME = "solidly_v2"
 VELODROME_V2_NAME = "velodrome_v2"
 
 SOLIDLY_FORKS = [AERODROME_V2_NAME, VELOCIMETER_V2_NAME, SCALE_V2_NAME, VELODROME_V2_NAME]
 
+
+def get_fee_1(address: str, is_stable: str, factory_contract: Contract):
+    return factory_contract.caller.getFee(address)
+
+
+def get_fee_2(address: str, is_stable: str, factory_contract: Contract):
+    return factory_contract.caller.getRealFee(address)
+
+
+def get_fee_3(address: str, is_stable: str, factory_contract: Contract):
+    return factory_contract.caller.getFee(address, is_stable)
+
+
+SOLIDLY_EXCHANGE_INFO = EXCHANGE_INFO
```

Comment:
I don't understand where all of this (those 3 functions, as well as the "upper-case" variable which follows them) is being used.
I also don't understand the point in assigning `SOLIDLY_EXCHANGE_INFO = EXCHANGE_INFO`.
Why not just use `EXCHANGE_INFO` as is (in case it is actually used anywhere)?

Reply:
Sorry for the nitpicking, but I now get the rationale of referring to it with a different name (though I'm not sure that this is indeed what you had in mind when you did so).

The cleanest solution here is probably to import it `as SOLIDLY_EXCHANGE_INFO`, and then use the latter.

Reply:
Yes I wanted to make it clear that it was exchange info only related to Soldily. That is much cleaner - I've updated it to this format.